Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -98,6 +98,23 @@ public extension Array where Element: Equatable {
         }
     }
 
+    /// SwifterSwift: Remove all duplicate elements removed using KeyPath to compare.
+    ///
+    /// - Parameter path: Key path to compare, the value must be Equatable.
+    /// - Returns: an array of unique elements.
+    @discardableResult
+    mutating func removeDuplicates<E: Equatable>(keyPath path: KeyPath<Element, E>) -> [Element] {
+        var items = [Element]()
+        removeAll { (element) -> Bool in
+            if !items.contains{ $0[keyPath: path] == element[keyPath: path] } {
```

Comment:
```suggestion
            if !items.contains { $0[keyPath: path] == element[keyPath: path] } {
```

Reply:
I think with early returns we usually do `guard`, no?
```Swift
guard items.contains { $0[keyPath: path] == element[keyPath: path] } else { return true }
```

Reply:
+1